UniversalAlgebra/Structures/Terms/Basic.lagda

```diff
@@ -46,11 +48,18 @@ Thus interpretation of a term is defined by structural induction.
 
 \begin{code}
 
-module _ {𝐹 𝑅 : signature}{χ : Level}{X : Type χ} where
+private variable
```

Comment:
It's common practice to put all the `private variable` declarations at the top, right after all the `open import`.

Reply:
Okay, but in this case I wanted to make clear that none of the variables in this large collection is used to define the inductive type of terms... but only for defining notation for interpretation of a term in a structure.

Reply:
But that's how variables work: they are only inserted by Agda when they are actually needed. So if they don't occur in the definition (or are shadowed), then they are not used.

The other option is to split this module into smaller pieces. The raw definitions go in one, the added utilities in another.

Reply:
Yes, I see.  However, here I want to make sure the user (not Agda) sees that this plethora of variables are not used in the relatively simple definition of Term.  So I think it makes sense, in this specific instance, to move the variables elsewhere.  On the other hand, I agree that it's probably better, then, to put all those variables and the definitions in which they are used, in another module.  (Feel free to make a PR to do that. :)